### PR TITLE
Send node instances

### PR DESCRIPTION
**Pull request by nktsitas**
Adds CloudCommand on top of agent-connection branch

### CODE REVIEW

**Review by papazach — APPROVED** [review]

**Review by chrikar — APPROVED** [review]

File: proto/agent/v1/connection.proto [hunk 1]
```diff
@@ -29,6 +30,19 @@ message UpdateAgentConnection {
     google.protobuf.Duration agent_uptime = 16;
 }
 
+message CloudCommand {
+    MessageType type = 1;
+    google.protobuf.Any payload = 2;
+}
```
Comment (lvrach) [inline proto/agent/v1/connection.proto:36]:
```suggestion
```

As we discussed, for now, there is no reason to wrap messages sent to the agent. The message type will be embedded in the message topic.

File: proto/agent/v1/connection.proto [hunk 2]
```diff
@@ -29,6 +30,19 @@ message UpdateAgentConnection {
...
+
+message SendNodeInstances {
+    string claim_id = 1;
+    string mqtt_broker_addr = 2;
+}
+
+enum MessageType {
+    MESSAGE_TYPE_SEND_NODE_INSTANCES = 0;
+}
```
Comment (lvrach) [inline proto/agent/v1/connection.proto:45]:
```suggestion
```

**Review by lvrach — CHANGES_REQUESTED** [review]
Minor changes, removal of the `CloudCommand` wrapping message.

File: proto/agent/v1/connection.proto [hunk 3]
```diff
@@ -29,6 +30,19 @@ message UpdateAgentConnection {
...
 }
 
+message CloudCommand {
+    MessageType type = 1;
+    google.protobuf.Any payload = 2;
+}
+
+message SendNodeInstances {
+    string claim_id = 1;
+    string mqtt_broker_addr = 2;
```
Comment (lvrach) [inline proto/agent/v1/connection.proto:40]: The `mqtt_broker_addr` is no longer neccessery for routing messages inside cloud.
```suggestion
    string mqtt_broker_addr = 2;
```

**Review by lvrach — APPROVED** [review]

**Pull request merged**